.github/workflows/build_debian_package.yml

```diff
@@ -55,18 +55,16 @@ jobs:
           (( $(curl -s -o /dev/null -w "%{http_code}" http://localhost:3000/ready/) == 200 )) || { echo "The HTTP request to the frontend fails"; exit 1; }
           (( $(curl -s -o /dev/null -w "%{http_code}" http://localhost:8000/ready) == 200 )) || { echo "The HTTP request to the backend fails"; exit 1; }
           docker ps
-          (cd smoke-tests;docker build -t smoke-tests .;)
-          sleep 5
-          docker run -t --net=host curlimages/curl:8.8.0 -s -o /dev/null -w "%{http_code}" http://localhost:8000/ready
-          echo
-          docker run -t --net=host curlimages/curl:8.8.0 http://localhost:8000/ready
-          echo
-          sleep 5
-          /usr/share/openhexa/openhexa.sh -g logs
-          sleep 5
-          # docker run -t --net=host smoke-tests https://app.demo.openhexa.org/ smoke-tests@bluesquarehub.com fekwus-moxmec-riFra1
-
-          docker run -t --net=host smoke-tests http://localhost:3000/ root@openhexa.org root || echo fail
-          sleep 5
-          /usr/share/openhexa/openhexa.sh -g logs
-          /usr/share/openhexa/openhexa.sh -g stop
+      - name: Install dependencies
+        working-directory: ./smoke-tests
+        run: npm ci
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+        working-directory: ./smoke-tests
+      - name: Run Playwright tests
+        run: npx playwright test
+        working-directory: ./smoke-tests
```

Comment:
can you explain me what have you dropped the container approach?

Reply:
I wanted to try with the default option proposed by Playwright when using GitHub Actions. I'll try with the docker image and see if it's working as well